generic/tests/cfg/netperf.cfg

```diff
@@ -71,14 +73,18 @@
     #    log_guestinfo_script = scripts/rh_perf_log_guestinfo_script.sh
     #    log_guestinfo_exec = bash
     #    log_guestinfo_path = /tmp/log_guestinfo.sh
+        server_tuned_profile = "tuned-adm profile virtual-guest"
         server_mtu_cmd = "ifconfig %s mtu %s"
     Windows:
     #    log_guestinfo_script = scripts/rh_perf_log_guestinfo_script.bat
     #    log_guestinfo_exec = cmd /c
     #    log_guestinfo_path = C:\log_guestinfo.bat
         server_mtu_cmd = "netsh interface ipv4 set subinterface "%s" mtu=%s"
+        i386, x86_64:
+            cpu_model_flags = ",hv_time,hv_relaxed,hv_vapic,hv_spinlocks=0xfff"
     client_mtu_cmd = "ifconfig %s mtu %s"
     host_mtu_cmd = "ifconfig %s mtu %s"
+    env_setup_cmd = "systemctl stop firewalld.service || service iptables stop ; echo 2 > /proc/sys/net/ipv4/conf/all/arp_ignore ; echo 0 > /sys/kernel/mm/ksm/run ; echo 0 > /proc/sys/kernel/watchdog ; echo 0 > /proc/sys/kernel/nmi_watchdog ; setenforce 1"
```

Comment:
This line is too long, could you use +=?

Reply:
I have updated, please help review. thanks